clang/include/clang/Driver/Options.td

```diff
@@ -5186,6 +5187,10 @@ def mrelax_all : Flag<["-"], "mrelax-all">, Group<m_Group>,
   Visibility<[ClangOption, CC1Option, CC1AsOption]>,
   HelpText<"(integrated-as) Relax all machine instructions">,
   MarshallingInfoFlag<CodeGenOpts<"RelaxAll">>;
+def msse2avx : Flag<["-"], "msse2avx">, Group<m_Group>,
```

Comment:
m_x86_Features_Group

Reply:
Never mind. It's an incorrect suggestion. `m_x86_Features_Group` is for feature after -mattr=

Reply:
This needs `TargetSpecific` Flags so that `--target=aarch64 -msse2avx` will error